packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -682,7 +726,7 @@ export class MgtPerson extends MgtTemplatedComponent {
       const text = this.getTextFromProperty(person, this.line1Property);
       if (text) {
         details.push(html`
-          <div class="line1" aria-label="${text}" @click=${() => this.handleLine1Clicked()}>${text}</div>
+          <div class="line1" aria-label="${text}">${text}</div>
```

Comment:
looks like you removed the click handles for line clicked, I assume that was unintentional?

Reply:
Yup sorry, fixed here:
3524f5a